discord/app_commands/tree.py

```diff
@@ -1076,6 +1077,17 @@ async def set_translator(self, translator: Optional[Translator]) -> None:
             await translator.load()
             self._state._translator = translator
 
+    async def _get_payload(self, *, guild: Optional[Snowflake] = None) -> List[Dict[str, Any]]:
+        commands = self._get_all_commands(guild=guild)
```

Comment:
Is the ordering of this iterator deterministic? If not, this needs a sort like the implementation of this we use in Salamander and Basilisk.